exonum-java-binding-cryptocurrency-demo/src/main/java/com/exonum/binding/cryptocurrency/HistoryEntitySerializer.java

```diff
@@ -16,18 +16,21 @@
 
 package com.exonum.binding.cryptocurrency;
 
+import static com.exonum.binding.common.serialization.StandardSerializers.protobuf;
 import static com.google.protobuf.ByteString.copyFrom;
 
 import com.exonum.binding.common.crypto.PublicKey;
 import com.exonum.binding.common.hash.HashCode;
 import com.exonum.binding.common.serialization.Serializer;
 import com.exonum.binding.cryptocurrency.transactions.TxMessageProtos;
 import com.google.protobuf.ByteString;
-import com.google.protobuf.InvalidProtocolBufferException;
 
 public enum HistoryEntitySerializer implements Serializer<HistoryEntity> {
   INSTANCE;
 
+  private static final Serializer<TxMessageProtos.HistoryEntity> historyEntityProtobufSerializer =
```

Comment:
As Serializers do not have side effects, they are considered to be 'constants', hence, must use screaming snake case. Also, as scope is small, I'd consider a shorter name (PROTO_SERIALIZER), though this is good too.